Scheduler: add Customize Toolbar demo description (#29869)
DevExpress/DevExtreme: apps/demos/Demos/Scheduler/ToolbarCustomization/description.md

> DevExtreme Scheduler includes a toolbar that can display predefined and custom items. You can specify the order and the position of toolbar controls. This demo illustrates how to implement a custom toolbar.

Reviewer: A couple of things that make me suggest an edit: 

- "order and position" - I don't see any difference between those two. 
- You start with "predefined and custom items", you finish with "custom toolbar". So what makes this toolbar custom? The fact that it contains custom items? Or maybe it has some other functionality? If it's just custom items, then I wouldn't call it a "custom toolbar". You already said that "a toolbar can display custom items".  

```suggestion
DevExtreme Scheduler can display a toolbar. You can populate the toolbar with predefined commands and custom items, in any order. This demo implements an example of toolbar customization.
```

Author: Yeah on second thought "To customize the toolbar..." is far superior to "To display a custom toolbar...". I have one question about "predefined and custom items", though: isn't it easier to read than "predefined commands and custom items"?
```suggestion
DevExtreme Scheduler can display a toolbar. You can populate the toolbar with predefined and custom items, in any order. This demo implements an example of toolbar customization.
```
If you think we should add something after "predefined", maybe we can use "controls" instead? They're called "predefined controls" in the Scheduler toolbar.items docs.
```suggestion
DevExtreme Scheduler can display a toolbar. You can populate the toolbar with predefined controls and custom items, in any order. This demo implements an example of toolbar customization.
```

Reviewer: Yeah - "predefined or custom items" works. So does your other version. You can pick the one you like the most.

<!--split-->

> To display a custom toolbar, add desired items to the [toolbar](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/toolbar/).[items[]](/Documentation/25_1/ApiReference/UI_Components/dxScheduler/Configuration/toolbar/items/) array. Scheduler supports the following types of toolbar items:

Reviewer: ```suggestion
To customize the toolbar, add items to the [toolbar](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/toolbar/).[items[]](/Documentation/25_1/ApiReference/UI_Components/dxScheduler/Configuration/toolbar/items/) array. Scheduler supports the following types of toolbar items:
```

- Predefined Controls    
    * dateNavigator    

>     Features arrows and a date interval button for switching dates. You can customize the elements to display and their order within the date navigator. 

Reviewer: ```suggestion
    Displays next/previous buttons and a date interval button that invokes a dropdown calendar. You can customize button availability and order. 
```

    * viewSwitcher    

>     Switches between view types (day, week, month, and others).

Reviewer 2: When you say `and others`, what do you mean? Year?

Author: Yeah. There are quite a few other view options available as well (http://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/), so I thought I should list only a few of the most important ones.

>     Switches between views like day, month, and week.

Reviewer: ```suggestion
    Switches between view types (day, week, month, and others).
```

    * today    

>     A "Today" button focusing the current date.

Reviewer: ```suggestion
    A "Today" button that navigates to the current date.
```

- DevExtreme Components    

> You can configure a DevExtreme component within a toolbar item element. In this demo, we extended the toolbar with a Button and TagBox.

Reviewer 3: TagBox -> SelectBox

Reviewer 3: new version of demos here https://github.com/DevExpress/DevExtreme/pull/29835

> You can configure a DevExtreme component within a toolbar item element. In this demo, we extended the toolbar with a Button and a TagBox.

Reviewer 2: ```suggestion
You can configure a DevExtreme component within a toolbar item element. In this demo, we extended the toolbar with a Button and TagBox.
```

> Configure the desired DevExtreme component within a toolbar item element. In this demo, we extended the toolbar's item collection with a Button and a TagBox.

Reviewer: ```suggestion
You can configure a DevExtreme component within a toolbar item element. In this demo, we extended the toolbar with a Button and a TagBox.
```
